docs/tutorials/python/linear-regression.md

```diff
@@ -94,32 +89,31 @@ lro = mx.sym.LinearRegressionOutput(data=fully_connected_layer, label=Y, name="l
 
 The above network uses the following layers:
 
-1. FullyConnected: The fully connected symbol represents a fully connected layer
+1. `FullyConnected`: The fully connected symbol represents a fully connected layer
    of a neural network (without any activation being applied), which in essence,
    is just a linear regression on the input attributes. It takes the following
    parameters:
 
-   - data: Input to the layer (specify the symbol whose output should be fed here)
-   - num_hidden: Number of hidden dimension which specifies the size of the output of the layer
+   - `data`: Input to the layer (specifies the symbol whose output should be fed here)
+   - `num_hidden`: Number of hidden dimensions which specifies the size of the output of the layer
 
-2. Linear Regression Output: Output layers in MXNet aim at implementing a
-   loss. In our example, the Linear Regression Output layer is used which
-   specifies that an l2 loss needs to be applied against it's input and the
-   actual labels provided to this layer. The parameters to this layer are:
+2. `LinearRegressionOutput`: Output layers in MXNet aim to implement a
+   loss. In our example, the `LinearRegressionOutput` layer specifies that an l2 loss
+   needs to be applied against it's input and the labels provided to this layer. The 	parameters to this layer are:
 
-   - data: Input to this layer (specify the symbol whose output should be fed here)
-   - label: The training label against whom we will compare the input to the layer for calculation of l2 loss
+   - `data`: Input to this layer (specifies the symbol whose output should be fed here)
+   - `label`: The training labels against which we will compare the input to the layer for calculation of l2 loss
 
-Note for Naming Convention: the label variable's name should be the same as the
-label_name parameter passed to your training data iterator. The default value of
-this is 'softmax_label', but we have updated it to lin_reg_label in this
-tutorial as you can see in Y = mx.symbol.Variable('lin_reg_label') and
-train_iter = mx.io.NDArrayIter(..., label_name="lin_reg_label")**
+**Note on naming convention:** the label variable's name should be the same as the
+`label_name` parameter passed to your training data iterator. The default value of
+this is `softmax_label`, but we have updated it to `lin_reg_label` in this
+tutorial as you can see in `Y = mx.symbol.Variable('lin_reg_label')` and
+`train_iter = mx.io.NDArrayIter(..., label_name='lin_reg_label')`.
 
-Finally, the network is stored into a *Module*, where you define the symbol
-who's value is to be minimised (in our case, lro or the lin_reg_output"), the
+Finally, the network is input to a *Module*, where you define the symbol
```

Comment:
Let's (almost) never use the pronoun "you". Instead, favor the scientific usage of "we" meaning the authors and reader, together. 

Change to:
"Finally, we give the network as input to a *Module*, where we define..."